clients/admin-ui/cypress/e2e/config-wizard.cy.ts

```diff
@@ -1,7 +1,7 @@
 import { stubSystemCrud, stubTaxonomyEntities } from "cypress/support/stubs";
 
 // skipping while configWizardFlag exists
-describe.skip("Config Wizard", () => {
+describe("Config Wizard", () => {
```

Comment:
you'll probably have to put the `skip` back in to make CI happy :(

Reply:
Oh, ha, yup